main.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'app'
+def main
+  app = App.new
+  app.run
+end
```

Comment:
- [ ] As stated in the project requirements your `main.rb` file should have a method that:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

- [ ] Currently, these are all written in the `app.rb` file. Kindly fix this 🙏